### DISCUSSION TITLE
plot label correlogram of the test set

### DISCUSSION THREAD

**Discussion by mar-du** [ultralytics/yolov5#8190] [Q&A]
Hi there,
I would like to report my label distribution of the test set.  I tried to input the labels of the test data in the plot_labels function but I get an input error. How should I transform the labels to input in the plot_labels function?
And do I understand it right that the label correlogram of the train.py script just shows the labels of the training data?
thanks a lot.

**Comment by glenn-jocher** [comment]
@mar-du yes train set is plotted by default. You might try switching your train and test paths in your data.yaml just to plot your test data

**Comment by mar-du** [comment]
Well, why not. Thanks! Otherwise I would have to use the Dataloader in the same way right? I thought about implementing the plot label function in the val.py but there I got the error...
